.changes/next-release/bugfix-AWSS3-54f8f8c.json

```diff
@@ -0,0 +1,6 @@
+{
+    "category": "AWS S3", 
```

Comment:
Technically the CRT is only exposed (in S3) via transfer manager. Can you change this to "Amazon S3 Transfer Manager"?